python/paddle/fluid/tests/unittests/test_imperative_resnet.py

```diff
@@ -286,6 +286,8 @@ def test_resnet_float32(self):
                 optimizer.minimize(avg_loss)
                 resnet.clear_gradients()
 
+                fluid.default_main_program().global_block()._clear_block()
```

Comment:
what's the plan to hide this call?

Reply:
After add the support of backward_hooks, this call could be removed